src/checks/company/relevant-countries-all.ts

```diff
@@ -0,0 +1,20 @@
+import { CompanyCheck } from '../../types/checks';
+
+const check: CompanyCheck = {
+    id: 'relevant-countries-all',
+    desc: 'If `relevant-countries` contains `all`, it should only contain `all`.',
+    severity: 'ERROR',
+    run: (json) => {
+        if (json['relevant-countries']) {
+            const all_and = json['relevant-countries'].includes("all") && json["relevant-countries"].length > 1
+            if (all_and)
+                return {
+                    message: `Record has redundant relevant-countries: \`all\` already covers : ${json["relevant-countries"].filter(x => x !== "all").join(",")}.`,
+                    json_pointer: '/relevant-countries',
+                    suggestions: ["all"] // should this be an array of arrays?

```

Comment:
Yes. You provide an array of values that the `json_pointer` should be replaced with. 

This would produce:

```json
{
	"relevant-countries": "all"
}
```